### PR TITLE
Revise logging and prevent server from creating extra calibration files

### PR DESCRIPTION
**Pull request by btmorr**
## overview

This PR is pulling double-duty as a ticket.

Gets rid of the `opentrons.util.log` module, which was making logging configuration much more confusing, and revises logging statements to be more useful in figuring out what the robot is doing. Removes a migration function that was causing multiple backups of the config file to be created on every import of Robot. Relates to #1347 

## changelog

- (refactor) Remove `opentrons.util.log` to unify logging configuration
- (refactor) Modify nginx logging config to write access logs to file instead of stdout
- (refactor) Reduce the amount of abstraction/translation used when loading and saving config files
- (fix) Remove unneeded migration script for config files that was causing extra duplication
- (refactor) Modify start script to use index file to find config file (does not affect robot behavior--only logging)

## review requests

Tested on 🌔 🌔 , would appreciate a run on a different robot to confirm that it boots as expected

### CODE REVIEW

File: api/opentrons/robot/robot_configs.py [hunk 1]
```diff
@@ -120,18 +119,18 @@ def _get_default():
...
-                'single': (0.0, 0.0, 0.0),
-                'multi': (0.0, 0.0, 0.0)
+                'single': [0.0, 0.0, 0.0],
+                'multi': [0.0, 0.0, 0.0]
             },
             'left': {
-                'single': (0.0, 0.0, 0.0),
-                'multi': (0.0, 0.0, 0.0)
+                'single': [0.0, 0.0, 0.0],
+                'multi': [0.0, 0.0, 0.0]
```
Comment (Laura-Danielle) [inline api/opentrons/robot/robot_configs.py]: What was the purpose of changing from tuple -> list?

File: api/opentrons/robot/robot_configs.py [hunk 2]
```diff
@@ -145,11 +144,10 @@ def _get_default():
 def load(filename=None):
     filename = filename or get_config_index().get('deckCalibrationFile')
     result = _get_default()
-
+    log.debug("Loading {}".format(filename))
```
Comment (mcous) [inline api/opentrons/robot/robot_configs.py]: 🦌🐛

File: api/opentrons/server/rpc.py [hunk 3]
```diff
@@ -170,7 +170,7 @@ def task_done(future):
                 .format(traceback.format_exc()), e)
         finally:
             log.info('Closing WebSocket {0}'.format(id(client)))
-            client.close()
+            await client.close()
```
Comment (Laura-Danielle) [inline api/opentrons/server/rpc.py]: Do we need to flush the stream before closing, or does 'await' key word function the same way?

Comment (mcous) [inline api/opentrons/server/rpc.py]: Oh this change has been needed for some time hah

**Pull request merged**